src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+
+## Communicating Events and Actions with Text and Sound
+
+A large part of interactive content are the events that happen during and after a user performs an action. Often these occur as on-screen visuals, animations, and transitions. The following are some techniques for creating an equivalent experience for non-sighted learners:
+
+* Use ARIA live-regions to communicate changes to the interface.
+* Use ARIA `describedby` to write text descriptions of what has happened.
+* Consider use of various sounds to convey meaning. For example, to convey proximity of one object to another, use a tone of increasing pitch like a Theremin (visit <a href="https://en.wikipedia.org/wiki/Theremin" rel="nofollow" target="_blank" class="link-external">Theremin on Wikipedia</a>).
+    * To find out more about how sound can be used to enhance learning, see <a href="https://wiki.fluidproject.org/display/fluid/Floe+Sonification+Work" rel="nofollow" target="_blank" class="link-external">Floe Sonification Work on the Fluid Project Wiki</a>.
+
+##  Content Accessibility / Screen Readers
+
+Follow a good structure and use semantic markup:
+* <a href="http://webaim.org/techniques/screenreader/" rel="nofollow" target="_blank" class="link-external">WebAim.org - Designing for Screen Reader Compatibility</a>
+* [Inclusive Learning Design Handbook - Accessibility Principles](AccessibilityPrinciples.html)
+* Make sure to use HTML 5 semantic markup wherever possible: i.e. `<article>`, `<section>`, `<figure>`, `<aside>`, `<header>`, `<footer>`. For more information, see: <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element" rel="nofollow" target="_blank" class="link-external">https://developer.mozilla.org/en/docs/Web/HTML/Element</a>
+* Use WAI-ARIA to improve accessibility. See: <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA" rel="nofollow" target="_blank" class="link-external">Mozilla Developer ARIA - good resource for learning, examples, blogs</a>.
+
+## Visuals
+
+* Provide text descriptions for important visuals: images should have alt-text and longer text descriptions if necessary. Use `aria-describedby` or `longdesc` as required. See: <a href="http://www.w3.org/WAI/tutorials/images/complex/" rel="nofollow" target="_blank" class="link-external">http://www.w3.org/WAI/tutorials/images/complex/</a>
+* Content contained in the background (using background CSS property), :before or :after elements are ignored by screen readers. Do not put important content within these elements. Use standard elements like `<image>`, `<div>`, `<figure>`, etc. as needed.
+* Images which are in the background should be cosmetic and not critical to the outcomes. Use `<img>` and `<figure>` as necessary for important graphics / images.
```

Comment:
These two bullet points about content in the background seem to be saying the same thing?